File: project3/Team2/backend/src/main/java/bitc/full502/backend/dto/AgencyProductResponseDTO.java
// src/main/java/bitc/full502/backend/dto/AgencyProductResponseDTO.java
package bitc.full502.backend.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class AgencyProductResponseDTO {
    private int pdKey;          // 제품 키
    private String pdNum;       // 품번
    private String pdProducts;  // 제품명
    private int pdPrice;        // 가격
    private boolean isSelected; // 프론트에서 선택 여부
    private int stock;          // 재고
    private LocalDate apStore;  // 입고일
}

File: project3/Team2/android/app/src/main/java/bitc/fullstack502/finalproject/AgencyProductResponseDTO.kt
package bitc.fullstack502.finalproject

// AgencyProductResponseDTO.kt
data class AgencyProductResponseDTO(
    val pdKey: Int,
    val pdNum: String,
    val pdProducts: String,
    val pdPrice: Int
) {
    var quantity: Int = 0           // 사용자가 입력할 수량
    var total: Int = 0              // quantity * price
    var isSelected: Boolean = false // 선택 여부
}
